text/0000-fabric-website.md

```diff
@@ -0,0 +1,57 @@
+
+Other Hyperledger projects, such as [Sawtooth](https://sawtooth.hyperledger.org/), already have websites similar to what this RFC proposes, that collate all relevant resources into one place.  Adding a website with a similar purpose for Fabric will serve as a "front door" to the project, and give it a more approachable and inviting feel.
+
+This website is for any user of Hyperledger Fabric, whether they are a developer of smart contracts or client applications, a network operator, or someone wishing to get involved in the Fabric community. It is for both new users wishing to get started with Fabric and existing users wanting to have a single place to access the resources they use most.
+
+The expected outcome for this project is to have a fully functioning website for our users, which is continually being updated with the latest and best resources.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+The current proposed structure for the website is that it will be made up of four primary sections, separated by tabs. These sections are based on the different types of Fabric user that could use this website as a resource. The sections are:
+
+- An **Overview** tab, that will include general information about Hyperledger Fabric and what it is;
+- An **Operators** tab, that will include an explanation of the role of a network operator and links to resources this type of user might find most useful, such as the operations guide and deployment guide;
+- A **Developers** tab, that will include an explanation of the role of a developer and links to resources this type of user might find most useful, such as the VS Code extension and developer tooling, the commercial paper sample, and the latest chaincode and gateway SDK versions;
+- A **Community** tab, that will include links to the most relevant resources for contributing to Fabric, such as the main Fabric gitHub repository, the Jira dashboard, and the contributing guide.
+
+References and links common to all user types (StackOverflow, the Hyperledger Fabric wiki, social media profiles etc.) are included at the bottom of all tabs.
+
+By laying the content out in this way, it is hoped that users will be able to easily locate the documentation they need to successfully achieve their Fabric use case. 
+
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+The proposed website will be implemented using React. It will be deployed to GitHub pages, after which it will be accessible at the link hyperledger.github.io/fabric-website. After deployment, the URL https://fabric.hyperledger.org/ will also redirect to hyperledger.github.io/fabric-website, similar to the existing website for Sawtooth.
```

Comment:
Since it is github pages, it's not going to be on the same repository as the Fabric core (https://github.com/hyperledger/fabric), right? 

Can we make sure we limit the Fabric core repository to its intended role, to be the Fabric core? :-)

Reply:
@yacovm  a gh-pages site on the URL  `hyperledger.gituhub.io/fabric-website` can only be served from the `fabric-website` repository.

Reply:
https://github.com/hyperledger/fabric-website is served via [CNAME](https://github.com/hyperledger/fabric-website/blob/gh-pages/CNAME) as [fabric.hyperledger.org](https://fabric.hyperledger.org)